custom/icds_reports/mobile_views.py

```diff
@@ -0,0 +1,5 @@
+from corehq.apps.hqwebapp import views as hqwebapp_views
+
+
+def login(req, domain):
+    return hqwebapp_views.domain_login(req, domain, custom_template_name='icds_reports/mobile_login.html')
```

Comment:
Guessing these will get more complex than just this, but if not, you could also pass through `custom_template_name` in a `kwargs` dict in the `url(...)` call, meaning you wouldn't need to define a separate view function for that purpose.

Reply:
Ah good call. Will keep that in mind if we don't end up needing any other changes.